app/3.0/docs/devguide/style-shadow-dom.md

```diff
@@ -278,110 +302,121 @@ You can create **slots** in an element's template that are populated at runtime.
 
 The basic syntax for incorporating slotted content looks like this:
 
-[See it on Plunker](http://plnkr.co/edit/bNvOvQqCEmC4DaoeNtwZ?p=preview)
+custom-element.js { .caption}
 
-`x-foo.html` { .caption}
-```html
-<dom-module id="x-foo">
-  <template>
-    <h1>
-      <slot name="title"></slot>
-    </h1>
-  </template>
-  ...
-</dom-module>
+```js
+...
+static get template() {
+  return html`
+    <h1><slot name="title"></slot></h1>
+  `;
+}
+...
 ```
 
-`index.html` { .caption}
-```html
-<link rel="import" href="x-foo.html">
+index.html { .caption }
 
-<x-foo>
-  <span slot="title">I'm a heading!</span>
-</x-foo>
+```html
+<head>
+  <script type="module" src="custom-element.js"></script>
+</head>
+<body>  
+  <custom-element><span slot="title">I'm a heading!</span></custom-element>
+</body>
 ```
 
+[See it on Plunker](http://plnkr.co/edit/e6m48f?p=preview)
+
 To style slotted content, use the `::slotted()` syntax.
 
 **Note:** To work within the Shady CSS scoping shim limitations, and to ensure consistent cross-browser behavior, add a selector to the left of the `::slotted(.classname)` notation (for example, `p ::slotted(.classname)`.
 
 `::slotted(*)` selects all slotted content:
 
-[See it on Plunker](http://plnkr.co/edit/pb0D6r15jvvxYVWsZ95U?p=preview)
+custom-element.js { .caption}
 
-`x-foo.html` { .caption}
-```html
-<dom-module id="x-foo">
-  <template>
+```js
+...
+static get template() {
+  return html`
     <style>
       p ::slotted(*), h1 ::slotted(*) {
         font-family: sans-serif;
         color:green;
       }
     </style>
-    <h1>
-      <div><slot name='heading1'></slot></div>
-    </h1>
-    <p>
-      <slot name='para'></slot>
-    </p>
-  </template>
-	...
-</dom-module>
+    <h1><div><slot name='heading1'></slot></div></h1>
+    <p><slot name='para'></slot></p>
+  `;
+}
+...
 ```
 
-`index.html` { .caption}
+index.html { .caption }
+
 ```html
-<link rel="import" href="x-foo.html">
-<x-foo>
-  <div slot="heading1">Heading 1. I'm green.</div>
-  <div slot="para">Paragraph text. I'm green too.</div>
-</x-foo>
+<head>
+  <script type="module" src="custom-element.js"></script>
+</head>
+<body>  
+  <custom-element>
+      <div slot="heading1">Heading 1. I'm green.</div>
+      <div slot="para">Paragraph text. I'm green too.</div>
+    </custom-element>
+</body>
 ```
 
-[See it on Plunker](http://plnkr.co/edit/Xb4j1r4wEgGuyUM9huFV?p=preview)
+[See it on Plunker](http://plnkr.co/edit/jMjMAY?p=preview)
 
 You can select by element type:
 
-`x-foo.html` { .caption}
-```html
-<dom-module id="x-foo">
-  <template>
+custom-element.js { .caption}
+
+```js
+...
+static get template() {
+  return html`
     <style>
       h1 ::slotted(h1) {
         font-family: sans-serif;
-        color: green;
+        color:green;
```

Comment:
for consistency, there should be a space between the property and the value (`color: green;`) here and below (`color: blue;`).